esp/templates/users/profiles/studentinfo.html

```diff
@@ -36,7 +37,9 @@ <h3>Additional Student Information</h3>
     {% endif %}
     </div>
 </div>
+{% endif %}
 
+{% if form.graduation_year %}
```

Comment:
a) I don't really think we should allow removing graduation_year from student_reg since then new accounts wont have grades and wont be able to access programs... 
b) If i do remove it from student_reg, i get an error when I submit the profile form:
![image](https://user-images.githubusercontent.com/25392075/80314202-286d2900-87be-11ea-9405-68a771d3adc2.png)

Reply:
Oh, that's a good idea. Any other fields that fields that you think should be "undeletable"?

Reply:
I've added email to the list. Those are the only two that I think we NEED to have?